src/components/30-organisms/modal/index.js

```diff
@@ -47,7 +48,7 @@ class AXAModal extends InlineStyles {
             ? 'o-modal__container--small'
             : ''}"
         >
-          ${!this.forced
+          ${!this.forced && !this.noheader
```

Comment:
The many `this`es here and below are a kind of Don't-Repeat-Yourself violation and can be avoided by destructuring at the beginning of the render method:

`const {forced, noheader} = this;`

Reply:
Done